added a gulpfile to help work on CSS with reload

ofZach opened the pull request:
this adds a simple gulpfile to work on css with reload (while the sass is getting compiled as part of python script to make the web book, gulp / browser-sync is much faster for testing changes on style).  Requires gulp, browser sync, gulp-sass   (ie, npm install browser-sync gulp-sass)

bilderbuchi commented:
> this adds a simple gulpfile to work on css with reload (while the sass is getting compiled as part of python script to make the web book, gulp / browser-sync is much faster for testing changes on style). Requires gulp, browser sync, gulp-sass (ie, npm install browser-sync gulp-sass)

should at least some of that info go into the readme or into /info ?

ofZach commented:
good point!  added a small note.  This isn't a necessity for building the book so I've added it more as an addendum, everything else above can build the book, this is just useful for css work (if you change the scss file you can see a live change on the website using browser-sync)

ofZach commented:
I'm going to merge this since it doesn't really have an impact on anything anyone else is diong and is just a helpful tool for css work.  I'm working on the front page / TOC now and this helps a ton (otherwise, you need to wait for the whole book to build to see chance to the scss file)
